source/RouteViewer/RendererR.cs

```diff
@@ -934,7 +934,7 @@ private struct ObjectFace {
 			GL.LoadIdentity();
 			GL.MatrixMode(MatrixMode.Projection);
 			GL.PushMatrix();
-			//GL.LoadIdentity();
+			GL.LoadIdentity();
```

Comment:
No idea why this got commented out, or why it worked until openTK's version got bumped, but there we go....